.github/workflows/test.yml

```diff
@@ -50,3 +50,22 @@ jobs:
       - uses: Swatinem/rust-cache@v1
       - run: cargo check -p near-sdk --features unstable
       - run: cargo check -p near-contract-standards
+  audit:
+    name: Audit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v2
+      - name: Install Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
```

Comment:
Does it matter if we only use stable here vs also MSRV (1.56.0)?

Reply:
it doesn't affect this since it's just checking the dependencies in the audit registry and not compiling things